star_align.py

```diff
@@ -116,8 +158,11 @@ def xml2RGdict(xmlfile):
     optional = parser.add_argument_group("optional input parameters")
     optional.add_argument("--out", default="out.bam", help="Name of the output BAM file")
     optional.add_argument("--workDir", default="./", help="Work directory")
+    optional.add_argument("--metaData", default=None, help="File containing metadata for the alignment header")
```

Comment:
Can we change this to --metaDataTab, so if we add other forms of metaData input later, we can break them out by input type?